<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourSettingEditWidget</class>
 <widget class="QWidget" name="ContourSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="contoursCheckBox">
     <property name="text">
      <string>Draw contour lines</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Line width:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="widthUnitLabel">
     <property name="text">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="useColorMapRadioButton">
     <property name="text">
      <string>Use values of colormap</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="numberOfValuesRadioButton">
     <property name="text">
      <string>Specify division number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="numberOfValuesSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>contoursCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberOfValuesRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberOfValuesSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
